getting-started/scaling-out.md

```diff
@@ -0,0 +1,128 @@
+# Scaling out using Distributed Hypertables
+
+Hypertables are used to handle large amount of data by chunking it up
+in smaller pieces, allowing operations to execute efficiently. When
+the amount of data grows beyond what you can handle with a single
+machine, you can distribute the data over several machines by using
+*distributed hypertables*.
+
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of partitioning of the hypertable and
+distribute the hypertable partitions on *data nodes*.
+
+![Deployment diagram for distributed hypertables]()
+
+## Working with Data Nodes
+
+Data nodes act as the containers of the hypertable partitions and are
+necessary to be able to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that data nodes are added as objects locally and you should
```

Comment:
"Note that a data node is local object on the access node that points to another database, typically on a remote host. The host could, however, be the same as the access node; this allows setting up a distributed database on a single server for, e.g., testing purposes. To achieve scale-out, however, each data node should be their own physical server with separate CPU and disk resources."